.github/workflows/deploy-service.yml

```diff
@@ -36,11 +36,11 @@ jobs:
         with:
           terraform_wrapper: false
       - name: Authenticate to Google Cloud
-        uses: google-github-actions/auth@v0
+        uses: google-github-actions/auth@v1
```

Comment:
are you sure this upgrade still relevant?

Reply:
no  but i tested a dew workflow and it's working so I see no reason not to upgrade